portal/views/portal.py

```diff
@@ -136,6 +137,15 @@ def report_slow_queries(response):
         SQLALCHEMY_RECORD_QUERIES = True
 
     """
+    if current_app.config.get("LOG_SLOW_RESPONSES"):
```

Comment:
Would it make sense to move this to a decorator or separate function? Small amount of code, but might be nice to separate benchmarking code from mission-critical functionality

Reply:
that whole function is just for that, debugging slowness...